gms/impl/src/test/resources/corpGroupESAutocompleteRequest.json

```diff
@@ -7,7 +7,6 @@
   },
   "post_filter" : {
     "bool" : {
-      "disable_coord" : false,
```

Comment:
What is the new default value? Or is "disable_coord" deprecated in 7?

Reply:
> The disable_coord parameter of the bool and common_terms queries has been removed. If provided, it will be ignored and issue a deprecation warning.

https://www.elastic.co/guide/en/elasticsearch/reference/6.8/breaking-changes-6.0.html

Seems like it has just been removed.